mobile_app_connector/models/account_invoice.py

```diff
@@ -156,6 +134,30 @@ def action_invoice_paid(self):
             if invoice.invoice_type in ('gift', 'fund') and has_app \
                     and not invoice.avoid_thankyou_letter:
                 invoice.send_mobile_notification()
+
+            # when a contract exist before than this invoice created, so the invoice.origin contains sponsorship
+            if invoice.invoice_type == 'sponsorship' and 'sponsorship' not in invoice.origin:
```

Comment:
I think this is the test we are looking for if we want to check only for mobile app invoices.
```suggestion
            if invoice.invoice_type == 'sponsorship' and ('android' in invoice.origin or 'ios' in invoice.origin):
```